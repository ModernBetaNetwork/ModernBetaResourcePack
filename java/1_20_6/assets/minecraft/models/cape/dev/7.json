{
	"parent": "minecraft:cape/cape_7",
	"textures": {
		"0": "item/capes/dev",
		"particle": "item/capes/dev"
	}
}