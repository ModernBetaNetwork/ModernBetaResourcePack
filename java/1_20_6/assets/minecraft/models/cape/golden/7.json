{
	"parent": "minecraft:cape/cape_7",
	"textures": {
		"0": "item/capes/golden",
		"particle": "item/capes/golden"
	}
}